_posts/2018-10-31-cockpit-181.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Cockpit 181
+author: mvo
+date: 2018-10-31
+tags: cockpit linux
+slug: cockpit-181
+category: release
+summary: ...
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We
+release regularly.  Here are the release notes from version 181.
+
+### Graph improvements
+
+The appearance of memory and CPU graphs has been improved, and their
+colors are more accessible.
+
+![New memory graph](/images/memory-graph.png)
+
+Memory on Linux is often hard to understand and usually presented
+incorrectly in most tools. There are so many different terms, most of
+which do not matter to anyone who's not a kernel developer.
```

Comment:
I'd leave out this sentence, it's not really descriptive/specific and might even be interpreted as derogatory.

Reply:
Yeah, good point. :+1:

Reply:
Removed